src/server/game/Conditions/ConditionMgr.cpp

```diff
@@ -1003,7 +1004,7 @@ void ConditionMgr::LoadConditions(bool isReload)
             cond->ErrorTextId = 0;
         }
 
-        if (cond->SourceGroup)
+        if (cond->SourceGroup || cond->SourceType == CONDITION_SOURCE_TYPE_PLAYER_LOOT_TEMPLATE)
```

Comment:
why? I guess as 0 (Horde) would count as no SourceGroup

Reply:
Yes, the IF fails because sourcegroup = 0 gets interpreted as false